Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2052,6 +2058,8 @@ class BrowserViewController: UIViewController {
     
     private var duckDuckGoPopup: AlertPopupView?
     func presentDuckDuckGoCallout(force: Bool = false) {
+        if Preferences.DebugFlag.skipNTPCallouts == true { return }
```

Comment:
one side effect here, with this flag `on`, tapping on ddg logo in private mode does not show up